src/utils/analytics-get-player-options.js

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line complexity
+export const analyticsGetPlayerOptions = (playerOptions) => {
+  return {
+    showJumpControls: playerOptions.showJumpControls || null,
+    seekThumbnails: playerOptions.seekThumbnails || null,
+    aiHighlightsGraph: playerOptions.aiHighlightsGraph || null,
+    floatingWhenNotVisible: playerOptions.floatingWhenNotVisible || null,
+    hideContextMenu: playerOptions.hideContextMenu || null,
+    analytics: playerOptions.analytics || null,
+    cloudinaryAnalytics: playerOptions.cloudinaryAnalytics || null,
+    playedEventPercents: playerOptions.playedEventPercents || null,
+    ...(playerOptions.cloudinary ? {
```

Comment:
[important] i would create inline functions for these sub objects, this will make the code much easier to read and at the same time will allow you to get rid of the eslint disable at the top of the file.